plugin/blocks/flexwall-break/block.json

```
@@ -0,0 +1,8 @@
+{
+  "apiVersion": 2,
+  "title": "FlexWall Break",
+  "name": "flexwall/flexwall-break",
```

```suggestion
  "name": "flexwall/break",
```

Same here. Double namespacing.